### ISSUE TITLE
Why still use BatchNorm for spconv(embedding) and Pool/UnPool?

### ISSUE THREAD

**Issue by mikel-zhobro** [Pointcept/PointTransformerV3#173]
Can you sharee the logic why Batchnorm is still used in the first spconv layer and in the (un)Pool layers?

Is there anything against replacing this with a LayerNorm, GroupNorm?

PS: If I understand it right, for the SerializedPooling, the BatchNorm would build the statistics over all points in all batches.

**Comment by tim-kuechler** [comment]
In the paper (section 4.3 -> Pooling Strategy) they state that experimentally they found BatchNorm to perform better than LayerNorm.

**Comment by Gofinge** [comment]
And also in our recent paper (Sonata / Concerto), we found BN block scaling up the data scale. Scaling up training with PTv3 backend with all LN resulting a better performance.
